scripts/sdw-admin.py

```diff
@@ -116,18 +118,6 @@ def perform_uninstall(keep_template_rpm=False):
         print("Reverting dom0 configuration")
         subprocess.check_call(["sudo", "qubesctl", "state.sls", "sd-clean-all"])
         subprocess.check_call([os.path.join(SCRIPTS_PATH, "scripts/clean-salt")])
-        if not keep_template_rpm:
-            print("Uninstalling Template")
-            subprocess.check_call(
-                [
-                    "sudo",
-                    "dnf",
-                    "-y",
-                    "-q",
-                    "remove",
-                    "qubes-template-securedrop-workstation-buster",
-                ]
-            )
```

Comment:
Should this be removed, or updated to remove the Bullseye template?

Reply:
So what's happening here is that while the qubes-template-securedrop-workstation-bullseye rpm exists, it's not installed via DNF and doesn't show up when you list installed packages. As far as I can tell, the qvm-template tool installs the relevant template files but then discards the downloaded package, which saves space when compared to 4.0. So the _keep template_ functionality would have to be implemented in a different manner to avoid repeated downloads. I'll file a new ticket for this, but I don't think it's a problem for the first 4.1 release as it's a developer convenience feature